### PR TITLE
Respect use_paged_results configuration option

### PR DESCRIPTION
**Pull request by HarlemSquirrel**
This allows turning off paged search results using the global
configuration option.

### CODE REVIEW

File: lib/active_ldap/adapter/base.rb [hunk 1]
```diff
@@ -180,7 +180,8 @@ def search(options={})
         limit = options[:limit] || 0
         limit = nil if limit <= 0
         use_paged_results = options[:use_paged_results]
-        if use_paged_results or use_paged_results.nil?
+        use_paged_results = @use_paged_results if options[:use_paged_results].nil?
```
Comment (kou) [inline lib/active_ldap/adapter/base.rb:183]:
```suggestion
        use_paged_results = @use_paged_results if use_paged_results.nil?
```

**Comment by kou** [comment]
Thanks!

**Comment by HarlemSquirrel** [comment]
If you have a few minutes to make a new version that would be awesome!

**Comment by kou** [comment]
Done!

**Pull request merged**